src/use_case/create_event/CreateEventInputData.java

```diff
@@ -0,0 +1,121 @@
+package use_case.create_event;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * The input data for creating event that will be passed into the create event interactor.
+ */
+public class CreateEventInputData {
+    private final String ownerUsername;
+    private final String eventName;
+    private final String location;
+    private final String time;
+    private final String type;
+    private final String description;
+    private final Boolean privacy;
+    private final Boolean inviteOnly;
+    private final Integer capacity; //Text box for capacity will only allow numbers to be inputted.
+    private final Integer ageRestriction;
+    private final String sexRestriction;
+    private final ArrayList<String> invitedPeople;
+
+    private CreateEventInputData(String owner, String eventName, String location, String time, String type, String description,
```

Comment:
If you have time in future, consider refractor this part of the code with the builder pattern. Right now the constructor takes too many instance variables and it would look simpler with a Builder.